persistence-cassandra-3.11/src/test/java/io/stargate/db/cassandra/impl/CassandraPersistenceIT.java

```diff
@@ -32,15 +37,24 @@
 
   private static CassandraPersistence persistence;
   private static File baseDir;
+  private static Config config;
 
-  @BeforeAll
-  public static void createPersistence(ClusterConnectionInfo backend) throws IOException {
+  @BeforeEach
```

Comment:
`@BeforeAll` must be a static method and I could only use other static methods and variables there. I had to make `getCassandraConfigPath()` non-static so that I could override it in CassandraPersistenceWithConfigIT.
However, the default JUnit TestInstance is `LifeCycle.PER_METHOD`, which means that `@BeforeAll` essentially behaves like `@BeforeEach` and creates a new test instance per method.

Reply:
I just ran `CassandraPersistenceIT` in IntelliJ and it was ok (in the main codebase, not on this PR).

I hope you did not forget to add `-Dstargate.test.backend.use.external=true` to the test JVM when running with an manually managed C* backend?

Reply:
Yes, I passed that parameter when I was running integration tests with my own C* node. I faced a similar issue today, I'll update the PR and explain shortly